lib/daterangepicker/daterangepicker.timesupport.js

```diff
@@ -228,7 +225,7 @@
                 });
 
                 $('<label class="time-support__to">To</label>').insertBefore(plugin.endPanel.$input);
-                $('<span class="time-support__zone">(UTC)</span>').insertAfter(plugin.endPanel.$input);
+                $('<span class="time-support__zone">(UTC' + timezone + ')</span>').insertAfter(plugin.endPanel.$input);
```

Comment:
A timezone could also be in IANA format e.g. `Europe/Rome`. The solution would be to remove UTC here and leave it up to the caller to pass it.